packages/servers/express-todo/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@expressjs/perf-servers-express-todo",
+  "version": "1.0.0",
+  "description": "Todo API reference app for Express performance benchmarking",
+  "main": "server.js",
+  "scripts": {
+    "setup": "npm i",
```

Comment:
We can consider `npm ci` for these setup scripts, to improve immutability of example servers. I'd like sha of commit to be immutable with runtime reality.

Reply:
We cannot because it also accepts a set of overrides we want to apply over the top. I also do not want to be testing on pinned back deps deep in the tree and then requiring an update PR all the time. Does that make sense as a reason to not do that?

Reply:
Ah right the overrides being one of the main goals, forgot about that.

I'd still like to see commit sha === an immutable artifact before we tweak things, to isolate changes, but I'll have to think more about whether or not that even makes sense in this context.